pgtoolkit/ctl.py

```diff
@@ -246,3 +254,57 @@ def status(self, datadir: Union[Path, str]) -> Status:
         """
         cmd = [str(self.pg_ctl), "status"] + ["-D", str(datadir)]
         return Status(self.run_command(cmd).returncode)
+
+
+def num_version(text_version: str) -> int:
+    """Return PostgreSQL numeric version as defined by LibPQ PQserverVersion
+    >>> num_version('pg_ctl (PostgreSQL) 9.6.3')
```

Comment:
can you add a blank line after the first line? (in `num_dev_version` below as well)